src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -1002,10 +1000,14 @@ TEST(Aktualizr, FullMultipleSecondaries) {
 
   TemporaryDirectory temp_dir2;
   UptaneTestCommon::addDefaultSecondary(conf, temp_dir, "sec_serial1", "sec_hwid1");
-  UptaneTestCommon::addDefaultSecondary(conf, temp_dir2, "sec_serial2", "sec_hwid2");
 
   auto storage = INvStorage::newStorage(conf.storage);
-  Aktualizr aktualizr(conf, storage, http);
+  UptaneTestCommon::TestAktualizr aktualizr(conf, storage, http);
+  UptaneTestCommon::addDefaultSecondary(conf, temp_dir2, "sec_serial2", "sec_hwid2");
+  if (boost::filesystem::exists(conf.uptane.secondary_config_file)) {
```

Comment:
Why is this `if` in an unit test? Shouldn't it fail if the file does not exist?

Reply:
Ah I saw that a similar code lies in `uptane_common_test.h`. If I understand correctly, this is a test to verify that adding a secondary works after `aktualizr` has been instantiated.

If so, a small explanatory comment would be helpful.

Reply:
It is redundant in the test, we actually should verify if the secondary instantiated successfully.

It's required in uptane_common_test.h as this code is shared among many tests including those that does not use secondaries. I will add explanatory comment.